corehq/warehouse/models/staging.py

```diff
@@ -70,14 +76,23 @@ def field_mapping(cls):
         ]
```

Comment:
just to specify that the `last_modified ` referred to the model. while in staging tables it doesn't matter too much, but in dim and fact tables there's also a `last_modified` field for when that row was last modified vs. when the model was. thought it might be useful to know. could be a bit over engineered